pulp_rpm/app/migrations/0019_migrate_updatecollection_data.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 2.2.14 on 2020-08-03 19:21
+
+from django.db import migrations
+
+def rename_update_collections(apps, schema):
+    """Insure every UpdateCollection associated with an Advisory has a unique name"""
+    UpdateCollection = apps.get_model("rpm", "UpdateCollection")
+    UpdateRecord = apps.get_model("rpm", "UpdateRecord")
+    for advisory in UpdateRecord.objects.all():
+        names = {}
+        for collection in advisory.collections.all():
+            if collection.name in names.keys():
+                collection.name = "{}_{}".format(collection.name, names[collection.name])
+                collection.save()
+            else:
+                names[collection.name] = 0
+
+def delete_orphan_collections(apps, schema):
+    """Delete orphaned collections"""
+    UpdateCollection = apps.get_model("rpm", "UpdateCollection")
+    UpdateCollection.objects.filter(update_record=None).delete()
+
+def map_update_collection_to_update_record(apps, schema):
+    """Point every collection to its/a/single UpdateRecord"""
+    UpdateCollection = apps.get_model("rpm", "UpdateCollection")
+    for collection in UpdateCollection.objects.all():
+        # At this point we can assume There Will Be Only One
+        ur = collection.update_record.all().first()
+        collection._update_record = ur
+        collection.save()
```

Comment:
Could we maybe do a bulk_update outside of the for loop?

Reply:
done